bin/commands/internal/container/prestop/index.ts

```diff
@@ -0,0 +1,63 @@
+/*
+  This program and the accompanying materials are made available
+  under the terms of the Eclipse Public License v2.0 which
+  accompanies this distribution, and is available at
+  https://www.eclipse.org/legal/epl-v20.html
+ 
+  SPDX-License-Identifier: EPL-2.0
+ 
+  Copyright Contributors to the Zowe Project.
+*/
+
+import * as std from 'std';
+import * as fs from '../../../../libs/fs';
+import * as common from '../../../../libs/common';
+import * as node from '../../../../libs/node';
+import * as shell from '../../../../libs/shell';
+import * as config from '../../../../libs/config';
+import * as component from '../../../../libs/component';
+
+export function execute() {
+  node.requireNode();
```

Comment:
you may not want this right here. in the shell script, it was assumed most everything required node.
we see in the library file that the catalog thing uses curl.js... so why not put the requireNode() over *there*
i believe the requireNode() function caches the answer about if node exists, so if you call it multiple times it should not harm performance.